### ISSUE TITLE
Test runner reports failing tests as passed if actual or expected contains ansi color codes

### ISSUE THREAD

**Issue by bobisageek** [cursive-ide/cursive#2973]
### Problem Statement
Using the test runner, a test that has failed is reported as passing if the actual or expected values are strings with ansi color codes in them.

### Repro
I have this namespace in a scratch project using deps.edn:
```clojure
(ns cursive-colors-test
  (:require [clojure.test :refer [deftest is]]))


(defn red [s]
  (str "\u001b[31m" s "\u001b[0m"))

(deftest no-coloring
  (is (= "hi" "hello")))

(deftest color-in-both
  (is (= (red "hi") (red "hello"))))

(deftest color-in-expected
  (is (= (red "hi") "hello")))

(deftest color-in-actual
  (is (= "hi" (red "hello"))))
```

Executing these tests using `run-tests` reports that all 4 tests fail (as they should). Executing these tests using the test runner (i.e. by clicking the gutter icon to run tests in the NS) indicates that the tests involving color in the strings passed.
![image](https://github.com/user-attachments/assets/0b0e02b7-bd25-452b-b2be-db7d39d69db1)

The console output appears to somewhat indicate that the tests failed:
![image](https://github.com/user-attachments/assets/b79baba9-d737-4c2d-989a-d686fb893f4d)
... but perhaps that data structure isn't being read/interpreted correctly.

#### IntelliJ version
IntelliJ IDEA 2024.2.2 (Ultimate Edition)
Build #IU-242.22855.74, built on September 18, 2024
#### Cursive Version
1.14.0-eap1-2024.2

**Comment by cursive-ide** [comment]
This seemed to break the IntelliJ test reporter somehow, you're right that the messages were printed but I'm not sure why they weren't interpreted correctly. Anyway, fixed:
<img width="616" alt="Screenshot 2024-12-02 at 21 05 45" src="https://github.com/user-attachments/assets/7e2ec894-122b-45ac-9f30-81a733123147">

**Comment by cursive-ide** [comment]
Fix released in 2025.1 on 2025-04-16.
